test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -128,6 +129,7 @@ def test_permission_on_scheduler(self):
                             SCHED,
                             id="testDeleteSched",
                             runas=ROOT_USER)
+        print self.server.schedulers
```

Comment:
This looks like a debug statement left in.  If it isn't, use self.logger() instead of print.